barretenberg/cpp/src/barretenberg/proof_system/arithmetization/arithmetization.hpp

```diff
@@ -125,7 +125,7 @@ template <typename _FF> class Turbo : public Arithmetization</*NUM_WIRES =*/4, /
     };
 };
 
-template <typename _FF> class Ultra : public Arithmetization</*NUM_WIRES =*/4, /*num_selectors =*/12> {
+template <typename _FF> class Ultra : public Arithmetization</*NUM_WIRES =*/4, /*num_selectors =*/11> {
```

Comment:
Back to 11 because we removed q_elliptic

Reply:
yup